src/index.js

```diff
@@ -0,0 +1,11 @@
+import Game from './controllers/Game.js';
+
+class Index {
+  async start() {
+    const game = new Game();
+    await game.start();
+  }
+}
+
+const index = new Index();
+index.start();
```

Comment:
사소하지만, 꼭 class여야 할까? 라는 생각이 들었어요!
```suggestion
new Game().start();
```